storage/storagepb/kv.proto

```diff
@@ -35,6 +35,10 @@ message Event {
   // a delete/expire event contains the previous
   // key-value
   KeyValue kv = 2;
-  // watchID is the ID of watching this event is sent to.
-  int64 watchID = 3;
+}
+
+message WatchResponse {
```

Comment:
move this back to watcher.go. We only need proto when we want to send it over wire. This type is just an internal message between two pkgs inside on process.